cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporterConfig.java

```diff
@@ -27,6 +27,11 @@
   public static final String CRUISE_CONTROL_METRICS_TOPIC_AUTO_CREATE_CONFIG = "cruise.control.metrics.topic.auto.create";
   private static final String CRUISE_CONTROL_METRICS_TOPIC_AUTO_CREATE_DOC = "Cruise Control metrics reporter will enforce " +
       " the creation of the topic at launch";
+  public static final String CRUISE_CONTROL_METRICS_TOPIC_AUTO_CREATE_TIMEOUT_CONFIG = "cruise.control.metrics.topic.auto.create.timeout";
+  private static final String CRUISE_CONTROL_METRICS_TOPIC_AUTO_CREATE_TIMEOUT_DOC = "Timeout on the Cruise Control metrics topic creation";
```

Comment:
Nit: Can we update (1) the config name and (2) the corresponding static variables for config and its doc to represent the unit of the timeout? -- i.e. can we add `.ms` and `_MS` prefixes, respectively?